bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/handler/TradfriGatewayHandler.java

```diff
@@ -237,9 +240,13 @@ protected boolean obtainIdentityAndPreSharedKey() {
             logger.error("Illegal gateway URI '{}'", authUrl, e);
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getMessage());
         } catch (JsonParseException e) {
-            logger.warn("Invalid response recieved from gateway '{}'", responseText, e);
+            logger.warn("Invalid response received from gateway '{}'", responseText, e);
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                    String.format("Invalid response recieved from gateway '%s'", responseText));
+                    String.format("Invalid response received from gateway '%s'", responseText));
+        } catch (ConnectorException |IOException e) {
+            logger.warn("Error connecting to gateway ",e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                    String.format("Error connecting to gateway '%s'"));
```

Comment:
What should `%s` replaced with?

Reply:
Thanks for spotting this, I removed the placeholder.